backend/app/Entities/Visit.php

```diff
@@ -42,8 +44,6 @@ final class Visit extends Model
         'geo_position_id'
     ];
 
-    protected $with = ['session', 'page', 'visitor', 'geo_position'];
```

Comment:
why do you need to delete this?

Reply:
loads too much extra data when all i need are some fields on the visitor model